packages/viem-account-ledger/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@celo/viem-account-ledger",
+  "version": "0.0.1",
+  "description": "Helper library to make ledger<->viem interactions easier",
+  "type": "module",
+  "exports": {
+    ".": "./lib/index.js"
+  },
+  "types": "./lib/index.d.ts",
+  "author": "cLabs",
+  "license": "Apache-2.0",
+  "homepage": "https://docs.celo.org/developer/tools",
+  "repository": "https://github.com/celo-org/developer-tooling/tree/master/packages/viem-account-ledger",
+  "keywords": [
+    "celo",
+    "blockchain",
+    "viem",
+    "ledger"
+  ],
+  "scripts": {
+    "build": "yarn run --top-level tsc -b .",
+    "clean": "yarn run --top-level tsc -b . --clean",
+    "docs": "yarn run --top-level typedoc",
+    "test": "NODE_OPTIONS='--experimental-vm-modules' yarn run --top-level jest --runInBand",
+    "lint": "yarn run --top-level eslint -c .eslintrc.js ",
+    "prepublishOnly": "yarn build"
+  },
+  "peerDependencies": {
+    "viem": "2.x"
+  },
+  "dependencies": {
+    "@celo/base": "^6.1.0",
+    "@celo/ledger-token-signer": "^0.4.0",
+    "@celo/utils": "^7.0.0",
```

Comment:
possibly can be removed if indeed ensureleading0x is the only use

Reply:
we're also using `generateTypedDataHash` from that package but only in tests, so moved to devDeps